### PR TITLE
add info logs for every request and response

### PR DESCRIPTION
**Pull request by ashwinspg**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
 
 import (
-	"io"
-	"log"
+	"fmt"
 	"net/http"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	http_router "github.com/julienschmidt/httprouter"
```
Comment (suryapandian) [inline main.go:9]: Also, take a look at https://godoc.org/github.com/go-chi/chi

File: main.go [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
 )
 
 func main() {
-	helloHandler := func(w http.ResponseWriter, req *http.Request) {
-		io.WriteString(w, "Server is running...")
+	port := "8080"
```
Comment (suryapandian) [inline main.go:13]: Can try setting such infrastructure-related configs as an environment variable.
Can use the [os](https://golang.org/pkg/os/) to get environment variables  using the function `os.Getenv("VARIABLE_NAME")`

**Review by suryapandian — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (ashwinspg) [reply main.go:9]: Sure, I'll look into the router package you requested

File: main.go (hunk 2 above)
Comment (ashwinspg) [reply main.go:13]: I have raised separate pull request for this suggestion:
https://github.com/ashwinspg/explore-golang/pull/2

**Pull request merged**